doc/articles/uno-development/ways-to-contribute.md

```diff
@@ -68,7 +68,7 @@ Whatever you do, let us know [through Twitter](https://twitter.com/unoplatform)
 
 ## Contributing code
 
-The UWP framework and WinUI are pretty big, but many hands make light work. We welcome code and content contributions from the community, and the core team is more than happy to help new contributors ramp up and familiarize themselves with Uno's codebase.
+The WinUI are pretty big, but many hands make light work. We welcome code and content contributions from the community, and the core team is more than happy to help new contributors ramp up and familiarize themselves with Uno Platforms's codebase.
```

Comment:
```suggestion
The WinUI framework is pretty big, but many hands make light work. We welcome code and content contributions from the community, and the core team is more than happy to help new contributors ramp up and familiarize themselves with Uno Platform's codebase.
```

Reply:
@jeromelaban, @sasakrsmanovic do we need to remove UWP framework here? As we still have support for it in a way with some versions.

There are also other places that will need to be reviewed, here for example:
https://github.com/unoplatform/uno/blob/c1656f662cfaf7a7d5d8c7850ef453589f622278/CONTRIBUTING.md?plain=1#L15
(Related search for uwp: https://github.com/search?q=repo%3Aunoplatform%2Funo%20uwp%20path%3A*.md&type=code)

Reply:
Other needed changes will be done in following PRs as per feedback from @sasakrsmanovic